tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import audb
+import audeer
+import audformat
+import audiofile
+
+
+pytest.NAME = 'db'
+pytest.VERSION = '1.0.0'
+
+
+@pytest.fixture
+def db(tmpdir, scope='function'):
+    r"""Publish a test database.
+
+    The database will use ``pytest.NAME`` as name
+    and ``pytest.VERSION`` as version.
+    It will be published to a file-system repository
+    in a tmp folder.
+    ``audb.config.REPOSITORIES`` will be adjusted
+    to point to the repository
+    the database was published to.
+
+    """
+    db_path = audeer.mkdir(audeer.path(tmpdir, pytest.NAME))
+
+    db = audformat.Database(
+        name=pytest.NAME,
+        source='https://github.com/audeering/audbcards',
+        usage='unrestricted',
+        expires=None,
+        languages='eng',
+        description='Example database.',
+        author='H Wierstorf, C Geng, B E Abrougui',
+        organization='audEERING',
+        license=audformat.define.License.CC0_1_0,
+    )
+
+    # Misc table 'speaker'
+    db.schemes['age'] = audformat.Scheme(
+        'int',
+        minimum=0,
+        description='Age of speaker',
+    )
+    db.schemes['gender'] = audformat.Scheme(
+        'str',
+        labels=['female', 'male'],
+        description='Gender of speaker',
+    )
+    index = pd.Index(
+        [0, 1],
+        dtype='Int64',
```

Comment:
Doesn't change match for this case but if 64 bits aren't needed we can use a smaller dtype maybe ?
```suggestion
        dtype='Int8',
```

Reply:
We usually use `pandas.Int64` as it allows to have `NaN` as well, which is not supported by the normal integer class, see https://pandas.pydata.org/docs/user_guide/integer_na.html.

But you are right it is not needed here.